extensions/blocks/eventbrite/deprecated.js

```diff
@@ -0,0 +1,147 @@
+import { _x } from '@wordpress/i18n';
```

Comment:
Do you think you could move this into a `eventbrite/deprecated/v1/save.js`, to stay consistent with how we do it for other blocks?

Reply:
Done in adeb0ad